.gitignore

```diff
@@ -20,3 +20,4 @@
 /config/application.yml
 
 /.env
+/.byebug_history
```

Comment:
The file is still present on the repo. You need to delete it and commit the change if you want to ignore it in the future